packages/forms/resources/views/components/section.blade.php

```diff
@@ -1,13 +1,14 @@
 <div
     @if ($isCollapsible())
         x-data="{ isCollapsed: {{ $isCollapsed() ? 'true' : 'false' }} }"
+        x-init="$watch('$store.errors?.all', () => isCollapsed = $el.getElementsByClassName('filament-forms-field-wrapper-error-message')[0] ? false : isCollapsed )"
```

Comment:
Does this code need changing since #3065?

Reply:
Yes, this is the magic :), it opens all sections that contains field errors, not just the first one.
- `x-on:expand-concealing-component.window` only opens the first found section with field errors.
- `$watch('$store.errors?.all'` opens all other sections that contains errors
- then the page scrolls to the **first** section containing field errors.
- sections without errors, remains closed.

Reply:
So could you fix the merge conflict for these changes please?